sstables/sstables.cc

```diff
@@ -1410,7 +1410,7 @@ void sstable::write_filter() {
     write_simple<component_type::Filter>(filter_ref);
 }
 
-void sstable::maybe_rebuild_filter_from_index(uint64_t num_partitions) {
+void sstable::maybe_rebuild_filter_from_index(uint64_t num_partitions, sstring origin) {
```

Comment:
@lkshminarayanan Wouldn't is make more sense to set `_origin` on the write path, e.g. by passing it to `open_sstable` so it would be available throughout the writing process?

Reply:
yeah that sounds better. This PR merged though. I'll create a fix and submit it as a separate PR.

Reply:
@bhalevy Submitted a PR https://github.com/scylladb/scylladb/pull/19721